Evaluation/Input/evaluation_plan.json

```diff
@@ -180,85 +183,73 @@
             "Project": "Compound",
             "Name": "COMPOUND",
             "Type": "Compound",
-            "SectionId": 31
+            "SectionId": 0
         },
         {
             "Project": "Compound",
             "Name": "Tablet",
             "Type": "Formulation",
-            "SectionId": 31
+            "SectionId": 0
         }
     ],
     "Sections": [
         {
-            "Id": 1,
-            "Title": "1 Introduction",
+            "Reference": "introduction",
+            "Title": "Introduction",
```

Comment:
2 changes: 
* new concept of `Reference `instead of `Id `for the sections
* Removing the chapter number from the title, because RE enumerates all chapter automatically - this applies both for the sections from the qualification plan as well as for the subchapters from the static content!